### ISSUE TITLE
Segment without ground truth labels

### ISSUE THREAD

**Issue by nifla2000** [openmedlab/MedLSAM#12]
Is there a way to segment specific anatomical structures without ground truth labels? (Is it a must to have two .txt files?) I just wanted to use this to segment with pre-trained weights. 
Thanks

**Comment by LWHYC** [comment]
> Is there a way to segment specific anatomical structures without ground truth labels? (Is it a must to have two .txt files?) I just wanted to use this to segment with pre-trained weights. Thanks

Hello,

Yes, theoretically it is possible to segment specific anatomical structures without ground truth labels. We are planning to add this feature to our project soon. Thank you for your interest and stay tuned for updates!

**Comment by radhi-bhalerao** [comment]
Hi, is there any update on this? I am also interested in segmenting without ground truth labels
